pkg/pool-weighted/contracts/smart/IndexPool.sol

```diff
@@ -93,11 +92,44 @@ contract IndexPool is BaseWeightedPool {
         // Immutable variables cannot be initialized inside an if statement, so we must do conditional assignments
         _tokens = tokens;
 
-        for(uint i = 0; i < numTokens; i++){
+        for (uint256 i = 0; i < numTokens; i++) {
```

Comment:
`uint8` will be enough here, there are 100 tokens max

Reply:
thanks solhint <> prettier integration + formatOnSave :)